### DISCUSSION TITLE
Is it possible to define a depName of a monorepo release?

### DISCUSSION THREAD

**Discussion by ekristen** [renovatebot/renovate#14501] [Request Help]
This repo uses monorepo like tagging for all the charts released but I'm failing to find a way to define the depname appropriately so it finds the right tag for the right chart I want to renovate the version for ...

https://github.com/prometheus-community/helm-charts

Is it possible to do this? I tried looking through the docs and the discussions but couldn't find what I was looking for, very possible I used the wrong terms.

For example I want to find the release `kube-prometheus-stack` and renovate that specific version.

Thanks.

**Comment by rarkins** [comment]
I'm not a helm chart user and don't know what you mean just by looking at that repo. Can you describe in more detail exactly what you mean? (Or alternatively you can leave as is and wait in case any other user with more helm knowledge than me can answer, in case you think it's clear enough)

**Reply by ekristen** [reply]
Hey @rarkins the helm aspect doesn't matter. 

I'm trying to figure out if renovate supports pulling dependencies from a monorepo type setup. 

The GitHub release are 'depname-version' instead of just version. 

I know there are other go and node projects that use this model.

I'm using a regex manager to define the data source, depname, etc, to update a field already. 

Does that help?

**Reply by rarkins** [reply]
Thanks, I understand now. The answer is No, we don't support this now. Can you raise a feature request and we can discuss options for this?
